optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/persistence/ConferenceSchedulingGenerator.java

```diff
@@ -446,31 +454,75 @@ private void createTalkList(ConferenceSolution solution, int talkListSize) {
                     break;
                 }
             }
+            Set<String> requiredRoomTagSet = new LinkedHashSet<>();
+            Set<String> preferredRoomTagSet = new LinkedHashSet<>();
+            Set<String> prohibitedRoomTagSet = new LinkedHashSet<>();
+            Set<String> undesiredRoomTagSet = new LinkedHashSet<>();
+            setRoomTags(requiredRoomTagSet, preferredRoomTagSet, prohibitedRoomTagSet, undesiredRoomTagSet);
+            Set<String> mutuallyExclusiveTagSet = new LinkedHashSet<>();
+            if (random.nextDouble() < 0.025) {
+                mutuallyExclusiveTagSet.add(mutuallyExclusiveTagList.get(random.nextInt(mutuallyExclusiveTagList.size())));
+            }
+            Set<Talk> prerequisiteTalkCodeSet = new LinkedHashSet<>();
+            if (random.nextDouble() < 0.025) {
+                prerequisiteTalkCodeSet.add(talkList.get(random.nextInt(talkList.size())));
+            }
             talk.setContentTagSet(contentTagSet);
             talk.setSectorTagSet(sectorTagSet);
             talk.setLanguage("en");
             talk.setRequiredTimeslotTagSet(new LinkedHashSet<>());
             talk.setPreferredTimeslotTagSet(new LinkedHashSet<>());
             talk.setProhibitedTimeslotTagSet(new LinkedHashSet<>());
             talk.setUndesiredTimeslotTagSet(new LinkedHashSet<>());
-            talk.setRequiredRoomTagSet(new LinkedHashSet<>());
-            talk.setPreferredRoomTagSet(new LinkedHashSet<>());
-            talk.setProhibitedRoomTagSet(new LinkedHashSet<>());
-            talk.setUndesiredRoomTagSet(new LinkedHashSet<>());
-            talk.setMutuallyExclusiveTalksTagSet(new LinkedHashSet<>());
-            talk.setPrerequisiteTalkSet(new LinkedHashSet<>());
+            talk.setRequiredRoomTagSet(requiredRoomTagSet);
+            talk.setPreferredRoomTagSet(preferredRoomTagSet);
+            talk.setProhibitedRoomTagSet(prohibitedRoomTagSet);
+            talk.setUndesiredRoomTagSet(undesiredRoomTagSet);
+            talk.setMutuallyExclusiveTalksTagSet(mutuallyExclusiveTagSet);
+            talk.setPrerequisiteTalkSet(prerequisiteTalkCodeSet);
             talk.setFavoriteCount(random.nextInt(1000));
-            talk.setCrowdControlRisk(0); // Disabled for now: the unsolved schedules must have a feasible solution
```

Comment:
We disable crowd control because most conferences want the opposite (no 2 rockstar speakers at the same time)